exercises/accumulate/Hints.md

```diff
@@ -0,0 +1,4 @@
+## Hints
+This exercise requires you to process a collection of data. You can simplify your code by using LINQ (Language Integrated Query).
```

Comment:
I would rephrase this to note that this exercise requires you to write an extension method. The link being used is great.